src/main/java/ee/carlrobert/codegpt/toolwindow/chat/ChatToolWindowTabPanelEditor.java

```diff
@@ -102,6 +122,25 @@ private JPanel createHeaderComponent(String language) {
 
   private JPanel createHeaderActions() {
     var wrapper = new JPanel(new FlowLayout(FlowLayout.LEFT, 0, 0));
+    wrapper.add(new IconActionButton("Diff", AllIcons.Actions.DiffWithClipboard, new AnAction() {
+      @Override
+      public void actionPerformed(@NotNull AnActionEvent event) {
```

Comment:
Perhaps it would make sense to validate if the target code is selected before diffing? Otherwise, the diff operation may not work as expected.
Examples can be taken from the replace action.